External/FEXCore/Source/Common/SoftFloat-3e/extF80_roundToInt.c

```diff
@@ -96,6 +96,7 @@ extFloat80_t
         switch ( roundingMode ) {
          case softfloat_round_near_even:
             if ( !(sigA & UINT64_C( 0x7FFFFFFFFFFFFFFF )) ) break;
+            __attribute__((fallthrough));

```

Comment:
Since we are C++20 shouldn't this be `[[fallthrough]];`?

Reply:
I'd have used that, but this is in a C file

Reply:
Oh right. SoftFloat-3e problems...